contributing.md

```diff
@@ -119,11 +119,22 @@ make docs
 
 ## Releasing
 
-To make a new release, from the master branch:
-
-* Rebuild docs to make sure they're in sync
-* Update `CHANGELOG.md` and `version.json`
-
-```bash
-make release
-```
+Our current process for publishing a release of python-pachyderm consists of the following steps:
+
+- [ ] Ensure compatibility with the latest protos:
+  - [ ] Make sure `version.json` references the most recent compatible release of Pachyderm
+  - [ ] Run `make src/python_pachyderm/proto/v2` to pull the latest pachyderm code
+  - [ ] Run `make lint`
+  - [ ] Deploy the version of pachyderm (and retrieve the version of `pachctl`) that matches `version.json` (the latest compatible release) and run out test suite (see above)
```

Comment:
What is the "test suite (see above)" referring to?

Reply:
I think it was a copy/paste error. That text has been removed.